backend/src/apiserver/client/pod_fake.go

```diff
@@ -14,6 +15,14 @@ import (
 type FakePodClient struct {
 }
 
+func (FakePodClient) GetEphemeralContainers(string, v1.GetOptions) (*corev1.EphemeralContainers, error) {
+	return nil, nil
```

Comment:
Where is this used?
Shall we add
```
glog.Error("This fake method is not yet implemented.")
```
?

Reply:
Sounds good. I will check it out a bit more and come back.

Reply:
I don't think this is specifically used, instead it is needed for the FakePodClient to adhere to the interface of the go client for Kubernetes: https://github.com/kubernetes/client-go/blob/cf84c08bad11b3ad990800f50914f8114d28a6c3/deprecated/typed/core/v1/pod.go#L40